src/agent/thread_helper.cpp

```diff
@@ -394,6 +394,14 @@ otError ThreadHelper::PermitUnsecureJoin(uint16_t aPort, uint32_t aSeconds)
             }
         });
     }
+    else
+    {
+        otExtAddress steeringClearData;
```

Comment:
Is this otExtAddress typo?

Reply:
No, the OpenThread function signature is
```
void otThreadSetSteeringData(otInstance *aInstance, const otExtAddress *aExtAddress);
```

Reply:
then the variable name should be extAddress?